utility_network/display-utility-associations/src/main/java/com/esri/samples/display_utility_associations/DisplayUtilityAssociationsSample.java

```diff
@@ -47,25 +50,10 @@
 
 public class DisplayUtilityAssociationsSample extends Application {
   
-  //Feature server for the utility network
-  private static final String FeatureServerUrl = "https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer";
   private MapView mapView;
   private GraphicsOverlay associationsOverlay;
   private UtilityNetwork utilityNetwork;
```

Comment:
We can sort these alphabetically for some extra neatness:
```suggestion
  private GraphicsOverlay associationsOverlay;
  private MapView mapView;
  private UtilityNetwork utilityNetwork;
```